group-actors/src/main/java/org/sunbird/actors/BaseActor.java

```diff
@@ -43,7 +43,7 @@ public void onReceive(Object message) throws Throwable {
         onReceive(request);
         endTrace(operation);
       } catch (Exception e) {
-        logger.error("{} : message : {} {}", operation, e.getMessage(), e);
+        logger.error("{} : exception occurred: {} {}", operation, e.getMessage(), e);
```

Comment:
let this log "<operation> exception" - why that semicolon which is not there in start/endTrace.